js/src/types.ts

```diff
@@ -47,21 +48,21 @@ export type MessageTypes = {
 }
 
 export interface MessageMetadata {
-  id: string
-  threadId: string
+  threadId: string // technically this has to be an rfq id because that's how you start a thread
   parentId?: string
```

Comment:
could i suggest that you also yeet this change this line to `parentId?: string | null` ? i dunno why but typescript gets upset when it's exported out as a node_module

![image](https://github.com/TBD54566975/tbdex-protocol/assets/118510103/89befb09-ff11-43ea-8e6a-1223b40e5614)

Reply:
nvm you've changed this bit
```const parentId = opts.last?.id.toString() ?? undefined```